### PR TITLE
A number of changes for the upcoming Scala 2.13.0-M5.

### PR DESCRIPTION
**Pull request by sjrd**
The changes in this PR allow `testSuite/test` to pass with today's tip of the `2.13.x` branch in scala/scala, i.e., commit https://github.com/scala/scala/commit/1ef688a914f2052ef558596e3beef6403054dc9a.

I have also locally tested that `testSuite/test` still passes with `++2.13.0-M4`.

### CODE REVIEW

**Comment by sjrd** [comment]
Updated for https://github.com/scala/scala/commit/80ee48b1880c8f817581a90f483b394273c237bc. Also `compiler/test` works.

**Comment by SethTisue** [comment]
fyi @som-snytt for the imports changes

**Comment by som-snytt** [comment]
@SethTisue thanks, I had verified that the delta made sense to me. Thanks @sjrd

File: junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala [hunk 1]
```diff
@@ -388,7 +388,14 @@ class ScalaJSJUnitPlugin(val global: Global) extends NscPlugin {
               varargsModule,
               definitions.wrapVarargsArrayMethodName(definitions.ObjectTpe),
               Nil, List(array))
-          gen.mkMethodCall(definitions.List_apply, List(wrappedArray))
```
Comment (gzm0) [inline junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala]: In this commit's message, maybe change "resist" to "support"? It feels like it's easier to understand.

File: javalib/src/main/scala/java/math/Conversion.scala [hunk 2]
```diff
@@ -241,7 +241,7 @@ private[math] object Conversion {
         val prev = v
         v /= 10
         currentChar -= 1
-        result = (48 + (prev - v * 10)).toChar + result
+        result = (prev - v * 10).toInt.toString + result
```
Comment (gzm0) [inline javalib/src/main/scala/java/math/Conversion.scala]: Where did the 48 + go?

Comment (SethTisue) [reply javalib/src/main/scala/java/math/Conversion.scala]:
```
scala> 48.toChar.toString == 0.toInt.toString
res4: Boolean = true
```

File: junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala (hunk 1 above)
Comment (sjrd) [reply junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala]: Done. Updated and rebased.

**Pull request merged**